lib/config/executor/container_vm.go

```diff
@@ -108,7 +123,7 @@ type ContainerVM struct {
 // in that there is no process inherently associated - this is closer to a ThreadPool than a Thread and
 // is the owner of the shared filesystem environment. This is the guest visible complement to ContainerVM.
 type ExecutorConfig struct {
-	Common `vic:"0.1" scope:"read-only" key:"common"`
```

Comment:
@gigawhitlocks The original common is still there, used by other objects. We just need to define a separate `common` for `executorConfig`. The reason is explained in my PR ^^^